.gitignore

```diff
@@ -1,6 +1,6 @@
 # our stuff
-/logs/
-/pids/
+logs/
+pids/
```

Comment:
when the webapp tests in created_files/webapp are run, they also produce log and pid files